en/css/README.md

```diff
@@ -297,7 +297,7 @@ Look at the code we just pasted to find the places where we added classes in the
 
 Don't be afraid to tinker with this CSS a little bit and try to change some things. Playing with the CSS can help you understand what the different things are doing. If you break something, don't worry – you can always undo it!
 
-We really recommend taking this free online [Codeacademy HTML & CSS course](https://www.codecademy.com/tracks/web). It can help you learn all about making your websites prettier with CSS.
+We really recommend taking this free online [freeCodeCamp - Basic HTML & HTML5](https://learn.freecodecamp.org/). It can help you learn all about making your websites prettier with CSS.
```

Comment:
Is it true that the "Basic HTML and HTML5" course subsection "can help you learn all about making your websites prettier with CSS."? https://learn.freecodecamp.org/ seems to have a separate section (also part of the "Responsive Web Design Certification") about CSS. Should we recommend both subsections? Or even the whole "Responsive Web Design Certification" course?

Reply:
@das-g  I believe the core courses would be ideal for beginners and the topics covered in the section, but we can point to the Responsive Web Design Certification course for further study. Should I put this in README?

Reply:
Which one(s) is / are "core courses"?

Reply:
the main topics covered in the session are HTML and CSS, freeCodeCamp has the general course from the most advanced basics in Web Design

Reply:
Not quite sure I understand. How about mentioning both their HTML course and their CSS course? E.g. like this:

```suggestion
We really recommend taking the free online courses "Basic HTML & HTML5" and "Basic CSS" on [freeCodeCamp](https://learn.freecodecamp.org/). They can help you learn all about making your websites prettier with HTML and CSS.
```